Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/db/DatabaseClient.java

```diff
@@ -305,4 +305,19 @@ private String getLikePattern(final String searchTerm) {
           
         };
       }
+
+	@Override
+	public List<ConceptEntry> getAllConceptsByStatus(ReviewStatus status) {
+		return conceptEntryRepository.getConceptsForGivenStatus(status);
+	}
+	
+	@Override
+	public List<ConceptEntry> getAllConceptsByStatusPaginated(ReviewStatus status, Integer page, Integer pageSize) {
+		return conceptEntryRepository.getConceptsForGivenStatus(status, PageRequest.of(page, pageSize));
+	}
+	
+	@Override
+	public Integer getNumberOfConceptsByStatus(ReviewStatus status) {
+		return conceptEntryRepository.getConceptsForGivenStatus(status).size();
```

Comment:
this should use a count method to have the db do the counting and not load all objects into memory